bases/br_inep_indicadores_educacionais/fluxo_educacao_superior/table_config.yaml

```
@@ -0,0 +1,286 @@
+source_bucket_name: basedosdados-dev
+project_id_staging: basedosdados-dev
+project_id_prod: basedosdados-dev
+table_id: fluxo_educacao_superior # AUTO GENERATED
+dataset_id: br_inep_indicadores_educacionais # AUTO GENERATED
+
+url_ckan: https://basedosdados.org/dataset/br-inep-indicadores-educacionais # AUTO GENERATED
+url_github: https://github.com/basedosdados/mais/tree/master/bases/br_inep_indicadores_educacionais # AUTO GENERATED
+
+version: <1.0> # REQUIRED
```

```suggestion
version: v1.0 # REQUIRED
```

se tiver entre <> não aparece